crates/nargo_cli/src/cli/init_cmd.rs

```diff
@@ -88,6 +95,9 @@ compiler_version = "{CARGO_PKG_VERSION}"
     // This uses the `match` syntax instead of `if` so we get a compile error when we add new package types (which likely need new template files)
     match package_type {
         PackageType::Binary => write_to_file(BIN_EXAMPLE.as_bytes(), &src_dir.join("main.nr")),
+        PackageType::Contract => {
+            write_to_file(CONTRACT_EXAMPLE.as_bytes(), &src_dir.join("main.nr"))
```

Comment:
Is `main.nr` the default filename we want to use?

Reply:
Yeah, I think this makes the most sense.

Reply:
Sounds good. I wasn't sure if we should do something like `contract.nr` but the pattern has been `main.nr` until now. We can make a separate breaking change if we want to change it.